### PR TITLE
Support multiple sorting options in Tabs

### PR DESCRIPTION
**Pull request by siemiatj**
Related to #12209

### CODE REVIEW

File: frontend/src/components/window/Tab.js [hunk 1]
```diff
@@ -20,18 +20,14 @@ class Tab extends PureComponent {
     } = this.props;
 
     if (docId && queryOnActivate) {
-      const query = orderBy
-        ? (orderBy[0].ascending ? '+' : '-') + orderBy[0].fieldName
-        : '';
-
       if (singleRowView) {
-        fetchTab({ tabId, windowId, docId, query }).then((res) => {
+        fetchTab({ tabId, windowId, docId, query: orderBy }).then((res) => {
```
Comment (teosarca) [inline frontend/src/components/window/Tab.js:24]: pls consider renaming `fetchTab` query param to `orderBy`. Saw that there are just 3 places where it's used (including this one).

File: frontend/src/utils/index.js [hunk 2]
```diff
@@ -281,3 +273,17 @@ export function deepUnfreeze(obj) {
 export function leftTrim(str) {
   return str.replace(/^\s+/, '');
 }
+
+/**
+ * @method formatSortingQuery
+ * @summary format's the ordering parameters prefixing them with asc/desc sign
+ */
+export function formatSortingQuery(orderBy) {
+  if (orderBy && orderBy.reduce) {
```
Comment (teosarca) [inline frontend/src/utils/index.js:282]: is there any reason to check for `reduce` instead of `map` which is used 1 line below?

**Review by teosarca — CHANGES_REQUESTED** [review]
check my comments.
Also provide a GIF or something about how it works, i.e.
* tab initially loaded
* user clicks on a column to order it ASC or DESC
* user clicks it again to same column 
* user clicks to a different column

File: frontend/src/utils/index.js (hunk 2 above)
Comment (siemiatj) [reply frontend/src/utils/index.js:282]: My previous version used reduce here, but it doesn't really matter. Can be any of them as they're both `array`-specific

**Comment by siemiatj** [comment]
Unfortunately it's too long for a gif https://recordit.co/ho4kf2a48X

**Review by teosarca — APPROVED** [review]

**Pull request merged**